app/views/tag/_sorting.html.erb

```diff
@@ -7,7 +7,11 @@ line up nicely if the user is asked to login on the questions/index page -->
 
 <form id="tagform" class="form-horizontal" method="GET" action="/questions/tag/">
   <div class="input-group">
+<<<<<<< HEAD
```

Comment:
I think you have not completely resolved the conflicts.
Either you can create a new branch with the updated branch and commit your changes in a new pr and close this one
OR you can rebase this pr and remove all the conflicts.

Please remove these changes.